src/focus-trap-react.js

```diff
@@ -101,17 +114,25 @@ class FocusTrap extends React.Component {
       }
 
       if (prevProps.active && !this.props.active) {
-        // NOTE: we never let the trap return the focus since we do that ourselves
-        this.focusTrap.deactivate({ returnFocus: false });
-        if (this.returnFocusOnDeactivate) {
-          this.returnFocus();
-        }
+        const deactivate = () => {
+          // NOTE: we never let the trap return the focus since we do that ourselves
+          this.focusTrap.deactivate({ returnFocus: false });
+          if (this.returnFocusOnDeactivate) {
+            this.returnFocus();
+          }
+        };
+
+        deactivate();
```

Comment:
I originally also delayed the deactivation of the trap but then I realised that it wasn't necessary to delay it.

The `deactivate()` helps add a bit of clarity to the code in my opinion but doesn't change the functionality.

If you want this can be reverted.